terraform/groups/ecs-service/main.tf

```
@@ -117,7 +117,7 @@ resource "aws_security_group" "ecs_sg" {
 }
 
 resource "aws_lb_target_group" "ollama_target_group" {
-  name     = "${local.service_name}-${var.environment}-target-group"
+  name     = "${local.service_name}-${var.environment}-tg-grp"
```

The grp suffix is redundant, as tg is the standard abbreviation for "target group".

```suggestion
  name     = "${local.service_name}-${var.environment}-tg"
```